bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-dynamic-number",
-  "version": "1.9.0",
+  "version": "1.9.1",
```

Comment:
please change it to version 1.9.2 because I set 1.9.1 for your last changes. 
Otherwise it wont publish to bower.